packages/panels/resources/views/components/sidebar/group.blade.php

```diff
@@ -22,22 +45,38 @@ class="fi-sidebar-group flex flex-col gap-y-1"
                 x-transition:enter-end="opacity-100"
             @endif
             @class([
-                'flex items-center gap-x-3 px-2 py-2',
+                'flex items-center rounded-lg px-2 py-2',
+                'transition duration-75 hover:bg-gray-100 focus:bg-gray-100 dark:hover:bg-white/5 dark:focus:bg-white/5' => $hasUrl,
+                'bg-gray-100 dark:bg-white/5' => $active,
                 'cursor-pointer' => $collapsible,
             ])
         >
-            @if ($icon)
-                <x-filament::icon
-                    :icon="$icon"
-                    class="fi-sidebar-group-icon h-6 w-6 text-gray-400 dark:text-gray-500"
-                />
-            @endif
+            @if ($hasUrl)
+                <a
+                    {{ \Filament\Support\generate_href_html($url, $shouldOpenUrlInNewTab) }}
+                    class="{{ $labelClasses }}"
+                >
+                    @if ($icon)
+                        <x-filament::icon
+                            :icon="($active && $activeIcon) ? $activeIcon : $icon"
+                            :class="$iconClasses"
+                        />
+                    @endif
 
-            <span
-                class="fi-sidebar-group-label flex-1 text-sm font-semibold leading-6 text-gray-700 dark:text-gray-200"
-            >
-                {{ $label }}
-            </span>
+                    {{ $label }}
+                </a>
+            @else
+                <span class="{{ $labelClasses }}">
```

Comment:
I'd rather have a Blade conditional in the tag `<{{ $hasUrl ? 'a' : 'span' }}>` than duplicating its contents.

Reply:
Then for closing the tag too?

```html
<{{ $hasUrl ? 'a' : 'span' }}>
<icon/>
Label
</{{ $hasUrl ? 'a' : 'span' }}>
```

Reply:
Yep.

Reply:
okay

Reply:
@zepfietje done